bundle/manifests/enterprise.splunk.com_indexerclusters.yaml

```diff
@@ -1354,6 +1354,42 @@ spec:
                       More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
                     type: object
                 type: object
+              securityContext:
```

Comment:
Could you please schedule a meeting to review these changes? Any CRD changes should be thoroughly reviewed. For any feature or CRD changes, we require a Requirements Document or Design Document, which should be reviewed during the meeting before the PR is raised. We can create a spike to gather all the necessary requirements beforehand.